x-pack/test/apm_api_integration/tests/alerts/rule_registry.ts

```diff
@@ -263,25 +282,28 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         expect(createResponse.status).to.be.below(299);
 
         expect(createResponse.alert).not.to.be(undefined);
-
         let alert = await waitUntilNextExecution(createResponse.alert);
 
-        const beforeDataResponse = await es.search({
-          index: ALERTS_INDEX_TARGET,
-          body: {
-            query: {
-              term: {
-                [EVENT_KIND]: 'signal',
+        const { body: targetIndices } = await getAlertsTargetIndices();
+
+        try {
+          await es.search({
+            index: targetIndices[0],
+            body: {
+              query: {
+                term: {
+                  [EVENT_KIND]: 'signal',
+                },
+              },
+              size: 1,
+              sort: {
+                '@timestamp': 'desc',
               },
             },
-            size: 1,
-            sort: {
-              '@timestamp': 'desc',
-            },
-          },
-        });
-
-        expect(beforeDataResponse.body.hits.hits.length).to.be(0);
+          });
+        } catch (exc) {
+          expect(exc.message).contain('index_not_found_exception');
```

Comment:
same here, there's only an assertion when the request fails

Reply:
Yup I'll update this to fail if a 200 is received as well.

Reply:
Fixed here: 6c5e2bfaeb4bf6016ed016e37eb2c7cea7c6fce7